Lib/ufo2ft/util.py

```diff
@@ -485,34 +484,31 @@ def _loadPluginFromString(spec, moduleName, isValidFunc):
     return klass(**options)
 
 
-def overrideSourceNamesFromDesignSpace(font, source, inplace=False):
-    """Allow overriding the family and style name in a source font
+def overrideSourceNamesFromDesignSpace(font, designspace_entry, inplace=False):
+    """Allow overriding the family and style name in a font
     if its designspace entry specifies custom values"""
-    _source_font = font
+    font_class = font.__class__
```

Comment:
nit: you only use `font.__class__` inside the defcon-related if block, you could just move this line in there

Reply:
I mean, just under `if not inplace:`

Reply:
Fixed thanks.